Monasca unit test is failing

```
E0726 16:29:54.236503    7778 driver.go:46] Post http://unexisting.monasca.com/metrics: read tcp 172.17.9.138:37831->141.8.224.93:80: read: connection reset by peer
--- FAIL: TestMonascaUnhealthy (0.46s)
	Location:	monasca_client_test.go:115
	Error:		Should be false
		
	<autogenerated>:29: ✅	GetToken()
FAIL
FAIL	k8s.io/heapster/metrics/sinks/monasca	0.669s
```

--- a/metrics/sinks/monasca/monasca_client_test.go
+++ b/metrics/sinks/monasca/monasca_client_test.go
@@ -102,12 +102,11 @@ func TestMonascaHealthy(t *testing.T) {
 }
 
 func TestMonascaUnhealthy(t *testing.T) {
-	// TODO: reenable once #1232 is fixed
-	t.Skip("skipping test due to #1232")
+
 	// setup
 	ksClientMock := new(keystoneClientMock)
-	monURL, _ := url.Parse("http://unexisting.monasca.com")
-	sut := &ClientImpl{ksClient: ksClientMock, monascaURL: monURL}
+	nonReachableMonURL, _ := url.Parse("http://127.0.0.1:9")
+	sut := &ClientImpl{ksClient: ksClientMock, monascaURL: nonReachableMonURL}
 	ksClientMock.On("GetToken").Return(testToken, nil).Once()
 
 	// do

